keys/signing/EOSK1.pub

```diff
@@ -13,74 +12,41 @@ uavyFLHTHtQcJmK2bFaD5cHv2MQBKw+oZo80mk0447chBkAWolxFoPSlOE5h0KV8
 XpAe8GUBdxe5mu28ogCvNdoyDyUxd+HgrV+7+WQYGcA9T88KC8WM4vdpVcuzQRrf
 9NFvlZbGf1ZpWlCRzsg6kWU3TS3TGaz2ImZUVsLULDms6lDojaEezcHG9QARAQAB
 tDtFT1MgT1NUcmVlIFNpZ25pbmcgS2V5IDEgKEVPU0sxKSA8bWFpbnRhaW5lcnNA
-ZW5kbGVzc20uY29tPokCPgQTAQIAKAUCU3O7KAIbAwUJCWYBgAYLCQgHAwIGFQgC
-CQoLBBYCAwECHgECF4AACgkQngjY2roC/Eb7MQ/9G2lvlWUR1uiL4gDDDFuBfDau
-cuB/DYvdKZD6TX7lR90mMlUDHSf4MPQQKZL/f757O2f7UOMt2oPjktkcXsUOjhvQ
-PLjOVnzONq6VKkvxHigRNO/Q2oQUTSlVG3HnCdDoN4fYqrD19AdqFXL1fqx1B+KY
-Gn9AwNd6ux82cHZT0deMk4YAZiV/fdSx+uF6FXbgtl0SGIrpQfovy+Pd6CKlKdz3
-1QMZamlcG00mU9T3+Txw9p88rcP8SOv9v38CeyUHrvguByLfTO20tveB+g2n9HQd
-aTw/0vlGbWogOTtMKpyWK3kGxd0ILw+/fsr4Ivo+f3TfEpP+s8hmxzt45cqgKnbe
-sSNjBu+IKBJ8pVqkw3oq4q6NPLYp72fXuc2CYA+ZX8noLpDo3oC7rA/ooGE89n76
-Y084XGwBm3A0LjC3wGdiw6R+WplJFs4AKlWd6cB3MdnoDOIj9+T9vhczILod/UO9
-L4xnSzkf2qER4iZPWemP+849ktA/agg8DtQldU1l3EnK36tzQpdxAaKqCe0CgDWL
-8Um80iKrw5ecXWnKbHzwQbdrAty9x+0R7IsFrpnpLzK4adPLRZVkkdv4bf/ROwZo
-4BYYxyzjACX+pkyOdTM3LXnBdlIzuNWY/rlJDjWULkcnAQA8sKeLX0+XngWZZQnv
-ubGU/TcZ9qTqzeWQvT+JAhwEEAECAAYFAlN09WoACgkQ5YA3Pt4jttnCBA/6AyQH
-vkC0TAwA0m3TmIvfensqdfHA0578ytHG6Vklwms8oHeOrUKXjU+Dq9XRMdBOHvDg
-UnBYUX1jsH2D1UftI0aCHnVeDgHVihUa5peuRgD2X6u9z6DFs29J1ZLL4COml16N
-abY6PYidSxgf2yaBF5e2jTd/lclwoMr3KPeXJUT4nVebjX/zxOiXQhaYp8IYqeke
-EBq7le3rc0yFJO1lwmGOFoED1JEMxmap34/FYhPe7r8KVbMmFxYq3ogQz6dGN7db
-Hi7Un8Xg3GHbAbhHR7uge/B0jIDYhojkFNkzHp1Rin5/ECk0aoCddGvzTNGNiCg1
-rY+OPjmzkkEcSkqKq2zVhUpRb7PjWfC62Kin34oWPaA7z840GgRKHIVh1P8b6De7
-b7O4lVb7U1tpnZ+F8m0ozJk9YclblRW5ekLu7cuzkQCuMFHtdNtR3/tngdYHFyHH
-F6+8Bqr9RvBQqlGeg8NZzU3M3n/0IhI2tgSSMeetXzAqRjEZ74UyWNnNVNQsiXwe
-u77S2dhxEWNYvSiFOGJgi8yoijFkfxHaTYyR0Fg6gW5AJ64x5liWiLjlhcBj29oa
-mkzl02pFm2t7hetkc5DdWgvFCTAnNB3KhLZ4UcI1FJD/E7Fa+vlV2ho38ug85kqo
-4lCRFlebUMOWjxA2wzhlBkvGuTVR4bBMudOlxuiJAhwEEAECAAYFAlN093QACgkQ
-aBBvffTJ/Ji4tg//f93UCZ+fcsAdC43GjyWQi9S2OX3GijsczKgOPk8FpLzICIkB
-BE98S9xh/EbqFVssp1tthVg0gQypnEH30Ruk+twNSJ+pMQQ8HKcKT6QIhLefy2hC
-Sac41ryvk50ewPKSlYY3kVlR0BZa/XmQ9S8Neoz674FYpicLTOAF08kIDnooBUUm
-tuBjMTt+vB0F2F/gyVb9k9P1vyJp1BC6n6AiQmX5PQWZX02jP8b6A3t4uKv1jpjE
-OHfE7GUuSNKr52LcwbXJsO0dJX4F7EkPwfRmKpWf52Kh86AG3iGqEAkcUta7s10e
-8jupIM7FjHChYB7WJN2Msm+lAXJOZQYEsh7mQTnfWBp7izcF5O2AUQTvVvbRNccR
-sFFoCzRB2o8cF00Vb0HDzhIAEnpQ/mJe14wwAnPANCYgU7d+vAIw7Qhsy6L2at0H
-A1IN2L2U41aI2+oqkbjPzN3BHKXgaV1s2OlJc1JMFlXsHLpBYLVln799el8Skf26
-uUhshiT9d+ilfswu402q+/zEoZ62LTx9Dy7iJbC8QTe8unqfsnDtTqJjKW81kG4A
-uS0FnKlHIzxzp8TTLJF7JVorgTJXP0Z1s/3VnqJ90Li5R1agf0WetNd8ta5cYcrM
-a9ZxMxgi3hjaH3X7R/s3OX09Pq9mkwET0Si0efb9/9jksRIrhf+86dUvYACJAhwE
-EAECAAYFAlN095IACgkQ0JfDQOGWwYrf2BAAlG6WRga9024zIw0mcPY8d12PfXQg
```

Comment:
Any idea why the new key data is so much smaller?

Reply:
Total random jumping in but I'm guessing the key that got generated is not 4k length anymore as the original was or ECC is being used instead of RSA.

Reply:
It's because the old key was signed with our master keys as well as by Srdjan. I'm not sure those are actually useful. Hmm... gpg makes it really hard to see signatures...

Reply:
I think I'll sign with our master keys again since it does provide some way to validate the keys if anyone is so inclined.

Reply:
@dbnicholson Nice catch! The signatures are there mostly to ascertain that someone trusts these keys even if it's just the master keys and at the time mine - ideally I was thinking we could build-in soft key rotation tooling if we had a "CA" pgp key authority (master keys) rather than hard-replacement (like we are forced to do now) kind of like TLS CAs but it never got fleshed out. If you might opt for trying to get that figured out it doesn't hurt to tack those sigs on there.

Reply:
Hmm. None of the tools involved do any kind of web of trust validation - trusting a primary key is a 1 or a zero. We were thinking about implementing key updates (so new uids, expirys, subkeys, etc) so we could provision some signing subkeys in a HSM and rotate those periodically, and keep the primary key in the sock drawer. That probably achieves similar to what you were thinking (an offline root of trust being different from the live signing key) but within the bounds of what tools like ostree actually support. I fixed subkey verification for ostree because Flathub uses them.

Reply:
(Also, hi! How's it going?)

Reply:
@ramcq yeah the web-of-trust is what I had in mind originally but if there are other ways to soft-rotate the keys, 👍 

(also doing well here - just got done releasing v1 of [this thing](https://github.com/cyberark/secretless-broker/releases/tag/v1.0.0) today. how's it gooing on your side? :) )

Reply:
Well, I added the master key signatures anyways since that's what they had before, but I agree it offers no difference to the tools. It would only be for someone manually inspecting the keys.